docs/configuration.asciidoc

```diff
@@ -218,3 +249,10 @@ This table outlines the attributes and their corresponding permissions:
 The configurable parameters provided by the Elastic APM agent aim to help configuring common use cases in an easy way, in most of the cases it means to act as a facade between your application and the OpenTelemetry Java SDK that this agent is built on top.
 If your project requires to configure more advanced aspects of the overall APM processes, you could directly apply that configuration using the https://opentelemetry.io/docs/instrumentation/java/getting-started/[OpenTelemetry SDK], which becomes available for you to use within your project by adding the Elastic agent plugin, as explained in <<setup,the agent setup guide>>.
 Said configuration will be used by the Elastic agent for the https://opentelemetry.io/docs/concepts/signals/[signals] it sends out of the box.
+
+[float]
+=== Central configuration
+
+Some of the runtime configurable options are available through Kibana's https://www.elastic.co/guide/en/kibana/master/agent-configuration.html[central configuration].
+
+You can take a look at the available options https://github.com/elastic/apm/blob/main/specs/agents/mobile/configuration.md[here].
```

Comment:
Why aren't we documenting this in the docs? With the other agents, we have a `dynamic` badge that we add to any configurations that can be changed dynamically (like with central config). Here's an example: https://www.elastic.co/guide/en/apm/agent/java/current/configuration.html.

Reply:
Sounds good! I didn't think much of it other than just reusing what's available elsewhere, to keep it as a single source of truth. Though now that I give it another thought, maybe that single source of truth, at least for these config options, should be here instead 🤔 it should be easier to read by avoiding jumping back and forth across different places. I'll make the changes 👍

Reply:
Awesome, thanks! Let me know if you need help with anything.

Reply:
I've just made the update, I _think_ it should be alright, although there's one line that reads `[options="header"]` that I'm not sure if it's needed. That's the only part I wasn't sure about when looking at Java Agent's docs.

Reply:
Yup, looks good! The `[options="header"]` line bolds the top row of the table you created; I'd leave it there.